File: Cinema-back-end/src/main/java/com/app/cinema/Ticket/Ticket.java
package com.app.cinema.Ticket;

import java.math.BigDecimal;
import java.sql.Timestamp;

public record Ticket(
        Integer id,
        BigDecimal price,
        Integer seatId,
        String seat,
        String room,
        Integer showingId,
        Timestamp showing,
        String movie,
        String age,
        String language,
        Integer length_minutes
) {
}

File: Cinema-front-end/js/classes/ticket.js
export default class ticket {

    id = null;
    price = null;
    seat = null;
    showing = null;
    showingName = null;
    seatCode = null;
    seatRoom = null;
    date = null;
    movie = null;

    constructor(id, price, seat, showing,) {
        this.id = id;
        this.price = price;
        this.seat = seat;
        this.showing = showing;
    }
}